lib/bitvec.cpp

```diff
@@ -137,3 +137,12 @@ unsigned bitvec::ffz(unsigned start) const {
 #endif
     return rv;
 }
+
+bool bitvec::is_contiguous() const {
+    // Empty bitvec is not contiguous
+    if (popcount() == 0)
```

Comment:
I know you said you weren't optimizing for speed, but it's an easy win to save `popcount()` to a temporary variable instead of calling it twice. I think it's worth doing.